dashboard/src/modules/components/ComparisonComponent/ChartGallery.jsx

```diff
@@ -0,0 +1,85 @@
+import {
+  BarElement,
+  CategoryScale,
+  Chart as ChartJS,
+  Legend,
+  LinearScale,
+  Title,
+  Tooltip,
+} from "chart.js";
+import {
+  Card,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateVariant,
+  Gallery,
+  GalleryItem,
+} from "@patternfly/react-core";
+
+import { Bar } from "react-chartjs-2";
+import React from "react";
+import { useSelector } from "react-redux";
+
+ChartJS.register(
+  BarElement,
+  Title,
+  Tooltip,
+  Legend,
+  CategoryScale,
+  LinearScale
+);
+const EmptyStateExtraSmall = (props) => (
+  <EmptyState variant={EmptyStateVariant.xs}>
+    <div>{props.message}</div>
+    <EmptyStateBody>Benchmark type is currently unsupported!</EmptyStateBody>
+  </EmptyState>
+);
+const ChartGallery = () => {
+  const { chartData, unsupportedType } = useSelector(
+    (state) => state.comparison
+  );
+  return (
+    <>
+      {unsupportedType ? (
+        <EmptyStateExtraSmall message={unsupportedType} />
+      ) : (
```

Comment:
I'm still not really convinced about trying to decode the API error for "unsupported benchmark". Obviously you can't generate the chart if the API call failed: but that's equally true if it failed due to a connection error or an internal Quisby error. Really, if the `dataset_visualize` API didn't return `response.ok`, there's nothing you can do here but display an error. Why not just display the API error without trying to interpret it?

Also, you're never displaying the *rest* of the "unsupported benchmark" message, which identifies the benchmark *Pbench* sees. The current Quisby integration relies on Pbench Agent postprocessing that requires the use of a pbench benchmark wrapper script. It's possible to run a `uperf` benchmark without the `pbench-uperf` wrapper, but the postprocessing won't be done and the server won't recognize it as "uperf" or be able to isolate the data Quisby needs. For example, using `pbench-user-benchmark -- uperf` allows you to specify `uperf` options the wrapper doesn't support, or run with a different `uperf` version. (I'm not sure about `uperf`, but looking at the actual production Pbench server, I see this a lot with `fio`.) But because we won't have the postprocessing the benchmark type (`dataset.metalog.pbench.script`) is "pbench-user-benchmark" rather than "uperf". It may seem obvious to the user that they ran uperf and the visualization "should" work, so reporting that we identified it as "pbench-user-benchmark" (while not ideal) at least might give them a clue what's confused us.

We have a goal to be able to remove that postprocessing from the Agent side and handle any processing dynamically on the server side, which may eventually remove this problem.

For now ... well, I'm not going to reject the PR because of this, because we can always iterate later: but it doesn't seem ideal, either.